app/models/piece.rb

```diff
@@ -17,6 +17,35 @@ def obstructed?(destination_row, destination_column)
     fail 'Input is invalid!'
   end
 
+  def move_to!(destination_row, destination_column)
+    # byebug
+
+    if occupied_space?(destination_row, destination_column)
+      opponent = game.pieces.where(row: destination_row, column: destination_column)
+
+      if capturable?(destination_row, destination_column)
```

Comment:
Rubocop doesn't like this, it wants a guardclause. Any suggestions on modifying this?